lua/wire/wireshared.lua

```diff
@@ -1014,10 +1010,12 @@ end)
 
 -- Nan never equals itself, so if the value doesn't equal itself replace it with 0.
 function WireLib.clampForce( v )
+	local x, y, z = v:Unpack()
+
 	return Vector(
-		v[1] == v[1] and math.Clamp( v[1], min_force, max_force ) or 0,
-		v[2] == v[2] and math.Clamp( v[2], min_force, max_force ) or 0,
-		v[3] == v[3] and math.Clamp( v[3], min_force, max_force ) or 0
+		x == x and clamp( x, min_force, max_force ) or 0,
+		y == y and clamp( y, min_force, max_force ) or 0,
+		z == z and clamp( z, min_force, max_force ) or 0
```

Comment:
Pretty sure the nan check here isn't needed since clamp will fix it. Otherwise, clampPos needs a nan check too.

Reply:
Yes, but then the force if the force is nan, then it will be minimal instead of being equal to 0